libs/antlr-2.7.7/antlr/ANTLRException.hpp

```diff
@@ -9,6 +9,9 @@
  */
 
 #include <antlr/config.hpp>
+#ifdef __MINGW32__
+#include <cstring>
+#endif
```

Comment:
Can you move this into a separate commit because it isn't directly linked to the hide column feature?

Reply:
When I pulled the project on Windows 10 to build it, I had issues with strcpy, and in the build order, this was the first header that complained about strcpy_s issues.  I agree that it could be put in as a separate issue or commit.  The commit string does mention it, on the second line.